scania-digital-design-system/sdds
# fix(icon): dependency on icon package

**Describe pull-request**  
Updated package-lock to include dep on icon/typo/color

**Solving issue**  
Fixes: [CDEP-2709](https://tegel.atlassian.net/browse/CDEP-2709)

**How to test**  
1. Go to Storybook
2. Check in Icon
3. Make sure both component and Css Icon works.


[CDEP-2709]: https://tegel.atlassian.net/browse/CDEP-2709?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
